spec/rubocop/cop/rspec/rails/minitest_assertions_spec.rb

```diff
@@ -506,6 +506,98 @@
         expect(a).not_to be_empty
       RUBY
     end
+
+    it 'registers an offense when using `assert_true`' do
```

Comment:
this is currently within the context for `*_empty` - ideally each assertion class should have its own `context`

Reply:
```suggestion
  end

  context 'with boolean assertions' do
    it 'registers an offense when using `assert_true`' do
```